scripts/update-niv.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -p git jq gitAndTools.hub niv -i bash
+
+set -euo pipefail
+
+author="serokell-bot <operations+github@serokell.io>"
+
+pkgs() {
+    jq ".[] | .repo, .rev" nix/sources.json | tr -d \" | xargs -n2 echo
+}
+
+changes() {
+    diff -y --suppress-common-lines <(echo "$1") <(echo "$2") | awk '{print $1": "substr($2, 0, 7)"->"substr($5, 0, 7)}' | column -t
+}
+
+new_pr() {
+    hub pull-request --no-edit -r serokell/operations -m "Update dependencies [automatic]" -m "$1"
+}
+[ -d nix ]
+
+git checkout -B "automatic/update"
+
+niv init
+[ $(git diff | wc -l) -eq 0 ] || {
+    git add nix
+    git commit --author=$author -m "Update niv's sources.nix file [automatic]"
+}
+
+before="$(pkgs)"
+
+niv update
+
+after="$(pkgs)"
+
+[ $(git diff | wc -l) -eq 0 ] || {
+    git add nix
+    git commit --author=$author -m "Update dependencies with niv [automatic]" -m "$(changes "$before" "$after")"
+}
+
+
+[ $(git diff origin/master | wc -l) -eq 0 ] || {
+    git push --set-upstream origin automatic/update
+    hub pr list | grep "Update dependencies \[automatic\]" || new_pr "\`\`\`$(changes "$before" "$after")\`\`\`"
```

Comment:
It's ironic that you change for duplicate PR but not for duplicate branch :joy:

Reply:
Added `--force`